pysat/_instrument.py

```diff
@@ -2450,8 +2450,8 @@ def rename(self, var_names, lowercase_data_labels=False):
                         fdict[oname] = nname
                     else:
                         # not in standard or higher order variable name keys
-                        estr = ' '.join((oname, ' is not',
-                                         'a known variable.'))
+                        estr = ' '.join(('cannot rename', oname, 'because it',
+                                         'is not a variable in this dataset.'))
```

Comment:
Updating error message to be more consistent with the style from xarray for consistency of testing.